plugins/lookup/github_app_access_token.py

```diff
@@ -34,6 +34,11 @@
         - Alternatively, you can use PyGithub (U(https://github.com/PyGithub/PyGithub)) to get your installation ID.
         required: true
         type: str
+      private_key:
+        description:
+        - GitHub App private key in PEM file format as string.
+        - Either O(key_path) or O(private_key) must be specified.
+        type: str
```

Comment:
Sorry, I think we forgot to mention this so far:
```suggestion
        type: str
        version_added: 10.0.0
```